src/utils/tokens.ts

```diff
@@ -415,12 +415,12 @@ const tokens = {
       }
     }
 
-    const tokens = token !== null ? [token] : [];
+    const tokensArray = token !== null ? [token] : [];
 
     return {
       inputs,
       outputs,
-      tokens,
+      tokens: tokensArray,
```

Comment:
Why the name changes in this file?

Reply:
Because of the [`no-shadow`](https://eslint.org/docs/latest/rules/no-shadow) rule. Those names were already declared in a scope above this one, so they should not be used again.